test/read-file-sync.test.js

```diff
@@ -43,7 +43,7 @@ describe('+ readFileSync()', function () {
         jf.readFileSync(file)
       }, function (err) {
         assert(err instanceof Error)
-        assert(err.message.match(file))
+        assert(err.message.match(/somefile\.json/))
```

Comment:
Yes, I have also done that.

![image](https://cloud.githubusercontent.com/assets/946879/10733124/33bee82c-7bf5-11e5-9220-3df2e388cb2e.png)